pkg/operation/botanist/component/etcd/bootstrap.go

```diff
@@ -524,9 +532,6 @@ spec:
                   clientPort:
                     format: int32
                     type: integer
-                  defragmentationSchedule:
```

Comment:
Why was this removed?

Reply:
It was valid for compaction cronjob which was in last release version of Druid